app/(main)/settings/team/actions.ts

```diff
@@ -422,3 +424,51 @@ export async function getCurrentUserRole() {
 		};
 	}
 }
+
+export async function getAgentActivities({
+	limit = 50,
+}: { limit?: number } = {}) {
+	try {
+		const currentTeam = await fetchCurrentTeam();
+
+		const activities = await db
+			.select({
+				agentId: agents.id,
+				agentName: agents.name,
+				startTime: agentActivities.startedAt,
+				endTime: agentActivities.endedAt,
+				usedCharge: agentActivities.totalDurationMs,
+			})
+			.from(agentActivities)
+			.innerJoin(
+				agents,
+				and(
+					eq(agentActivities.agentDbId, agents.dbId),
+					eq(agents.teamDbId, currentTeam.dbId),
+				),
+			)
+			.orderBy(desc(agentActivities.startedAt))
+			.limit(limit);
+
+		const formattedActivities = activities.map((activity) => ({
+			...activity,
+			// Convert milliseconds to seconds and round to 2 decimal places
+			usedCharge: Math.ceil((activity.usedCharge / 1000) * 100) / 100,
```

Comment:
:memo: This calculation converts milliseconds to seconds and rounds up to 2 decimal places.

For example, if `activity.usedCharge` is 12345 milliseconds:

1. `activity.usedCharge / 1000`
   - 12345 / 1000 = 12.345 (Converting milliseconds to seconds)

2. `(activity.usedCharge / 1000) * 100`
   - 12.345 * 100 = 1234.5 (Shifting decimal point 2 places to the right)

3. `Math.ceil((activity.usedCharge / 1000) * 100)`
   - Math.ceil(1234.5) = 1235 (Rounding up decimal places)

4. `Math.ceil((activity.usedCharge / 1000) * 100) / 100`
   - 1235 / 100 = 12.35 (Shifting decimal point 2 places to the left)

Result:
- 12345 milliseconds → 12.35 seconds